service/src/main/java/uk/gov/hmcts/reform/fpl/service/respondent/RespondentValidator.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.fpl.service.respondent;
+
+import com.google.common.collect.ImmutableList;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.common.Party;
+import uk.gov.hmcts.reform.fpl.service.FeatureToggleService;
+import uk.gov.hmcts.reform.fpl.service.RespondentService;
+import uk.gov.hmcts.reform.fpl.service.ValidateEmailService;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static uk.gov.hmcts.reform.fpl.enums.State.OPEN;
+
+@Component
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
+public class RespondentValidator {
+
+    private static final int MAX_RESPONDENTS = 10;
+
+    private final RespondentService respondentService;
+    private final ValidateEmailService validateEmailService;
+    private final RespondentAfterSubmissionValidator respondentAfterSubmissionValidator;
+    private final FeatureToggleService featureToggleService;
+    private final Time time;
+
+    public List<String> validate(CaseData caseData, CaseData caseDataBefore) {
+        ImmutableList.Builder<String> errors = ImmutableList.builder();
+
+        validateMaximumSize(caseData).ifPresent(errors::add);
+
+        validateDob(caseData).ifPresent(errors::add);
+
+        List<Respondent> respondentsWithLegalRep =
+            respondentService.getRespondentsWithLegalRepresentation(caseData.getRespondents1());
+        List<String> emails = respondentService.getRespondentSolicitorEmails(respondentsWithLegalRep);
```

Comment:
check if everything that is after submission is allowed, and if everything is set for submission